auto-delegate-examples/build.gradle.kts

```diff
@@ -3,8 +3,8 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_15
-    targetCompatibility = JavaVersion.VERSION_15
+    sourceCompatibility = JavaVersion.VERSION_16
```

Comment:
Recommend upgrading to the `--relase` flag https://docs.gradle.org/current/userguide/building_java_projects.html#sec:java_cross_compilation

Reply:
Ah, good call. I flagged #5 so i don't forget about this before my next release. I got burned by this hard recently manifesting in: https://www.morling.dev/blog/bytebuffer-and-the-dreaded-nosuchmethoderror/